website/data/pages/principles.mdx

```diff
@@ -6,10 +6,9 @@ description: It's an open source design system for plugins, products and digital
 # Principles
 
 ![Strapi Design System](/strapi-design-system-cover.png)
-
 # What is Strapi Design System ? 
 
-It's an open-source design system for plugins, products and digital experiences. This design system consists of design principles and guidelines, created from our design vision, components, design tools and resources, and a vibrant community of contributors.
+It's an open-source Design System for plugins, products, and digital experiences. This design system consists of design principles and guidelines, created from our design vision, components, design tools and resources, and a vibrant community of contributors. Strapi Design System is the new version of Buffet.js.
```

Comment:
> Strapi Design System is the new version of Buffet.js.

Are we sure about this sentence? @soupette Wdyt about that?

Reply:
IMO the Design System isn't related to BuffetJS at all. Should we really quote BuffetJS?

Reply:
@maevalienard could you pls have a look

Reply:
I agree @HichamELBSI we should not mention Buffet.js

Reply:
We had a discussion regarding this point in a Slack thread here: https://strapihq.slack.com/archives/CEN0F9WF6/p1644587440299209

Reply:
Completely forgot this discussion, my bad. But, since it is not the new version of BuffetJS (technically), maybe we can mention that Strapi Design System replace the old BuffetJS library.

Reply:
Are you ok with this modification @philippkhon ?

Reply:
Sure, thanks @HichamELBSI

Reply:
@philippkhon Any update regarding this PR?

Reply:
@HichamELBSI I've just updated the last line accordingly